vprof/code_heatmap.py

```diff
@@ -91,7 +91,8 @@ def _calc_skips(self, heatmap, num_lines):
             skips.append((prev_line, num_lines - prev_line))
         return skips
 
-    def _skip_lines(self, src_code, skip_map):
+    @staticmethod
```

Comment:
AFAIK community prefers not to pass the object in static methods when not explicitly required.